.travis.yml

```diff
@@ -2,6 +2,8 @@ before_install:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS='-Xmx768m -XX:MaxPermSize=384m'"
   - "echo $GRADLE_OPTS"
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
 #  - "export GRADLE_OPTS='-Dorg.gradle.daemon=true'"
```

Comment:
Comments are bad in general (I will try to use daemon on Travis once again), but it would be good to keep two lines with GRADLE_OPTS together.